sdk/instrumentation/net/http/handler.go

```diff
@@ -45,16 +57,39 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		span.SetAttribute(key, value)
 	}
 
-	span.SetAttribute("http.url", r.URL.String())
+	url := r.URL.String()
+	span.SetAttribute("http.url", url)
+
+	// run url filters
+	for _, urlEvaluator := range h.urlFilters {
```

Comment:
All this filter logic should be encapsulated in one filter package which hides all the details.  Then each client (instrumentation lib) only needs to call one function rather than iterating over each filter, ie
`
if filter.evaluateURL(span, url) {
  w.WriteHeader(http.StatusForbidden)
  return
}
`

I think this is where you were also going right JC?

Reply:
I see, done.